pkgs/by-name/ge/gema/package.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchurl
+, installShellFiles
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gema";
+  version = "2.0";
+
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/gema/files/gema/gema-${finalAttrs.version}/gema-${finalAttrs.version}-source.tar.gz";
+    hash = "sha256-c7qxgZYk/QaqocjRXVlgJxUWCgf3T1JBY7v9Fg9YfIU=";
+  };
+
+  makeFlags = [ "--directory=src" ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 src/gema -t $out/bin
+    installManPage doc/gema.1
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "General purpose text processing utility based on the concept of pattern matching.";
+    homepage = "https://gema.sourceforge.net/";
+    license = lib.licenses.mit;
+    longDescription = "Reads an input file and copies it to an output file transforming the data as specified by the patterns defined by the user.";
```

My sincerest apologies for failing to communicate this better, @quag, `meta` attributes are not "strictly" sorted alphabetically as it does make much sense for `description` and `longDescription` to be next to each other.

```suggestion
    description = "General purpose text processing utility based on the concept of pattern matching.";
    longDescription = "Reads an input file and copies it to an output file transforming the data as specified by the patterns defined by the user.";
    homepage = "https://gema.sourceforge.net/";
    license = lib.licenses.mit;

```

---

Haha. Fixed! Thanks for all your help. Every time I submit a package I learn a lot from people like yourself. I appreciate it.

---

I'm merely passing on what I have learned from others who generously shared their knowledge and expertise.
It would make day and be helpful for project if you feel like doing the same for others 🙂

---

I would like to help. Is there a good way or place to start?

---

That's great to hear, @quag 😃

An excellent start is [`CONTRIBUTING.md`](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md). The nixpkgs repo also provides many README.md files, the ones I found interesting and insightful are [`pkgs/README.md`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md) and [`pkgs/by-name/README.md`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md). These READMEs contain links to other READMEs, so I think this should be plenty to dig into for a start. Feel to share interesting tidbits with me that you find, I'm always curious to hear what others come across.

I've made it an occasional habit to [look for new package pull requests](https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+is%3Aopen+is%3Apr+-is%3Adraft+is%3Aopen+-author%3Ar-ryantm+-label%3A%226.topic%3A+python%22+init) that seem a good fit for me to review—this is how I came across this PR 🙂 

Is this helpful for you and answers your question, @quag?